docs/app/Examples/Button/Animated.js

```diff
@@ -0,0 +1,15 @@
+import React, {Component} from 'react';
+import {Button} from 'stardust';
+
+export default class extends Component {
+  render() {
+    return (
+      <Button className='animated'>
+        <div className='visible content'>Next</div>
+        <div className='hidden content'>
+          <i className='right arrow icon'></i>
```

Comment:
I've noticed the linter half-assing on this as well. I only catch it because of my IDE.